package.json

```diff
@@ -57,5 +61,6 @@
     "fetch-everywhere": "^1.0.5",
     "query-string": "^6.2.0",
     "universal-url": "^2.0.0"
-  }
+  },
+  "typings": "build/index.d.ts"
```

Comment:
Hmmm, I know we ignore the generated Flow types, since we place them in the `build` dir, but it might be nice to have the TS types checked in, so we can see the diffs to types when we change things in Flow-land.

What do you think about packaging the TS types in a dir, like `typings/index.d.ts`? That way, we could validate changes to these types and know when we introduce a breaking change. This might require some CI effort to ensure we're keeping them up to date as well...

Reply:
Totally, my initial implementation was committing the type file. I added some basic check see a2b8170's commit message.